### DISCUSSION TITLE
kubernetes-dashboard-kong pod in CrashLoopBackOff after a clean install

### DISCUSSION THREAD

**Discussion by jesuslpm** [kubernetes/dashboard#8996] [General]
I have a clean Docker Desktop Kubernetes where I installed dashboard using:

```
helm repo add kubernetes-dashboard https://kubernetes.github.io/dashboard/
helm upgrade --install kubernetes-dashboard kubernetes-dashboard/kubernetes-dashboard --create-namespace --namespace kubernetes-dashboard
```

I then executed `kubectl get pod -n kubernetes-dashboard`:

```
NAME                                                  READY   STATUS             RESTARTS      AGE
kubernetes-dashboard-api-768f5bd99f-q4ns5             1/1     Running            0             84s
kubernetes-dashboard-auth-6445f5448f-9ddns            1/1     Running            0             84s
kubernetes-dashboard-kong-75bb76dd5f-qqllc            0/1     CrashLoopBackOff   3 (38s ago)   84s
kubernetes-dashboard-metrics-scraper-8d8778c4-htqz7   1/1     Running            0             84s
kubernetes-dashboard-web-7d77999479-g9k8t             1/1     Running            0             84s
```

And here is the output of `kubectl logs kubernetes-dashboard-kong-75bb76dd5f-qqllc -n kubernetes-dashboard`

```
Defaulted container "proxy" out of: proxy, clear-stale-pid (init)
Error: could not prepare Kong prefix at /kong_prefix: nginx configuration is invalid (exit code 1):
nginx: [warn] the "user" directive makes sense only if the master process runs with super-user privileges, ignored in /kong_prefix/nginx.conf:7
nginx: the configuration file /kong_prefix/nginx.conf syntax is ok
nginx: [emerg] bind() to [::1]:8444 failed (99: Cannot assign requested address)
nginx: configuration file /kong_prefix/nginx.conf test failed
```

**Comment by floreks** [comment]
Something occupies the port that kong is trying to use: 
`nginx: [emerg] bind() to [::1]:8444 failed`

**Reply by jesuslpm** [reply]
I have no idea of what it could be. I think is something internal in kubernetes-dashboard-kong pod.

**Reply by floreks** [reply]
It is not. It simply cannot bind to port 8444 as it is already occupied. I can't tell you what is using it. You'd have to debug it on your own.

**Reply by jesuslpm** [reply]
How can I debug it myself? as I understand the port is already bound in the container. The Kubernetes cluster is totally empty, just reset, so this should work without my intervention.

**Reply by floreks** [reply]
With docker desktop it is basically a port forward from the container to the host. Either docker desktop already uses it or something else on your host.

**Reply by jesuslpm** [reply]
I upgraded to the latest docker desktop and it works now

**Reply by karthikGenAI** [reply]
I get the same issue in GCP auto-pilot cluster any clues? @floreks  @jesuslpm

**Reply by floreks** [reply]
@karthikGenAI https://github.com/kubernetes/dashboard/issues/8765#issuecomment-1985546677
